src/pyvesync/vesync.py

```diff
@@ -83,7 +109,7 @@ def get_devices(self):
 
     def get_energy_dict_from_api(self, response):
         data = {}
-        data['energy_consumption_of_today'] = response['energyConsumptionOfToday']
+        data['energy_cons_of_today'] = response['energyConsumptionOfToday']
```

Comment:
was the shortening strictly for pep8? if so, i think the dictionary item is a little less clear with the change than it was. what do you think of the following instead?
```
    def get_energy_dict_from_api(self, r):
        d = {}
        d['energy_consumption_of_today'] = r['energyConsumptionOfToday']
```